### DISCUSSION TITLE
Google OAuth refresh token

### DISCUSSION THREAD

**Discussion by xingster** [fief-dev/fief#447] [Q&A]
I'm having an issue where logged-in users frequently get "OAUTH_PROVIDER_REFRESH_TOKEN_ERROR" when trying to retrieve their Google OAuth access token. I'm using offline_access scope.

I read that Fief supports refreshing the access token automatically using the endpoint below: curl \
-X GET \
-H 'Authorization: Bearer FIEF_ADMIN_API_KEY' \
'https://fief.mydomain.com/admin/api/oauth-providers/OAUTH_PROVIDER_ID/access-token/USER_ID'

**Comment by frankie567** [comment]
I added a warning log for this case: https://github.com/fief-dev/fief/pull/451

If you upgrade to the new version (when released), you'll be able to check this log so you and me will have more clues about what's going wrong :)
